src/algebra/char_p/basic.lean

```diff
@@ -338,6 +338,11 @@ lemma cast_eq_mod (p : ℕ) [char_p R p] (k : ℕ) : (k : R) = (k % p : ℕ) :=
 calc (k : R) = ↑(k % p + p * (k / p)) : by rw [nat.mod_add_div]
          ... = ↑(k % p)               : by simp [cast_eq_zero]
 
+end
+
+section
```

Comment:
Same here, doesn't seem worth it to make a `section` for a single `variable` for a single `theorem`.

Reply:
OK, but it feels weird to put a statement into a section that has `variables [ring R]` that does not talk about rings...
The file has some pretty short sections already.

Reply:
I'll remove the extra sections anyway, though.